testscript/pts/test-scripts/cast/cast-to-int.sqlts

```diff
@@ -35,27 +35,27 @@ for::{
         { value: "-9223372036854775808", result: -9223372036854775808 }, // min
 
         // float
-        { value: "`1.5e0`",  result: 1 },
-        { value: "`1.9e0`",  result: 1 },
+        { value: "`1.5e0`",  result: 2 },
+        { value: "`1.9e0`",  result: 2 },
```

Comment:
The `HONOR_PARAMETER` mode rounds half to even, https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/src/org/partiql/lang/eval/ExprValueExtensions.kt#L348